### ISSUE TITLE
Update `object-alt` rule description at Deque University

### ISSUE THREAD

**Issue by kantoche** [dequelabs/axe-core#3480]
Product: axe-core documentation

Expectation: Fix `object-alt` rule at https://dequeuniversity.com/rules/axe/4.4/object-alt according to the deletion of the inner text as an alternative to an object element (see #3318) 

Actual: Description still mentions :"Add alternative text to all embedded elements using **either inner text,** title attributes, aria-label or aria-labelledby" and the examples have not been updated.

Motivation: updated rule description

**Comment by WilcoFiers** [comment]
You're absolutely right. Thank you for reporting this. We'll be sure to get it fixed.

**Comment by yecril71pl** [comment]
The `title` attribute does not contain ~*alternative text*~, it contains *tool tip text*.  For example:
```html
<INPUT TYPE=SUBMIT VALUE="Get rich quick!"
 TITLE="
Press this button to get filthy rich in absolutely no time!
  (Rules and conditions apply.)" >
```
You do not want the title to be presented as alternative text for the button in this case.  It contains an *additional explanation* about the button that is presented to you when you focus on it.  It *refers* to the button, so the button must already exist in the first place.
Having a `title` attribute attached to every problematic element causes tool tips to appear over all of them for everybody, even if most of them do not require any tool tips to be presented to users who do not require assistance.  It gets very annoying and distracting. 
TL;DR: The ~`title`~ attribute should be removed too.

**Comment by yecril71pl** [comment]
How is this completed?

**Comment by yecril71pl** [comment]
Thank you.  I still do not get why we need this.
1. If the embedded object is a Web page, why does it need alternative text?
2. If the embedded object contains multimedia, why not use the alternative text in the metadata within the embedded object?
3. If the embedded object contains an applet, why not rely on the applet&rsquo;s accessibility?
